hypothesis-python/tests/nocover/test_floating.py

```diff
@@ -16,12 +16,12 @@
 """Tests for being able to generate weird and wonderful floating point numbers."""
 
 import math
-import sys
+from sys import float_info
```

Comment:
Let's keep `sys.float_info` in this file; consistency is a virtue but this PR is already a big diff.